src/repository/opamRepositoryBackend.mli

```diff
@@ -45,6 +45,7 @@ module type S = sig
       [checksum] can be used for retrieval but is NOT checked by this
       function. *)
   val pull_url:
+    ?full_fetch:bool ->
```

Comment:
It is missing documentation about the new argument.

Reply:
same as above.

Reply:
fair enough